priv/repo/migrations/20221031170029_add_test_group.exs

```diff
@@ -0,0 +1,38 @@
+defmodule Skate.Repo.Migrations.AddTestGroup do
+  use Ecto.Migration
+
+  def change do
+    create table(:test_groups) do
+      add(:name, :string, null: false)
+      timestamps()
+    end
+
+    create table(:test_groups_users) do
+      add(
+        :test_group_id,
+        references(:test_groups,
+          on_delete: :delete_all,
+          on_update: :update_all
```

Comment:
question (non-blocking): Is the `on_update` needed? What would be updated on this record if a test_group is updated?

Reply:
This made me curious so I dug up the [relevant section in the Postgres docs](https://www.postgresql.org/docs/13/ddl-constraints.html):

> Analogous to ON DELETE there is also ON UPDATE which is invoked when a referenced column is changed (updated). The possible actions are the same. In this case, CASCADE means that the updated values of the referenced column(s) should be copied into the referencing row(s).

If I'm reading this correctly, this basically means that this would trigger if `test_group.id` was updated and therefore any corresponding `test_groups_users.test_group_id` valyes needed updating to match the new value. I don't think we're going to go around changing the IDs of test groups but it seems like a harmless case to cover.